packages/components/src/advertisement/utils.jsx

```diff
@@ -0,0 +1,8 @@
+export const getOptions = (object = {}) => (
```

Comment:
`_.filter(object || {}, (val) => val)` ?

Reply:
only useful for an array of objects, we need to filter object keys 
<img width="326" alt="Screen Shot 2021-02-01 at 4 39 56 PM" src="https://user-images.githubusercontent.com/789031/106480883-269e6780-64ac-11eb-8ac4-ebbf0427052a.png">

Reply:
`_.pickBy(object || {}, (val) => val)`

Reply:
that could fit